autogpt_platform/backend/backend/api/features/chat/tools/get_doc_page.py

```
@@ -128,19 +142,19 @@ async def _execute(
         try:
             content = full_path.read_text(encoding="utf-8")
-            title = self._extract_title(content, path)
+            title = self._extract_title(content, params.path)
 
             return DocPageResponse(
                 message=f"Retrieved documentation page: {title}",
                 title=title,
-                path=path,
+                path=params.path,
                 content=content,
                 doc_url=self._make_doc_url(path),
```

Ruff (0.14.14)

[error] Undefined name `path`

(F821)